### PR TITLE
Added: routes and ui for home, signin and signup pages

### PR DESCRIPTION
**Pull request by pratik-kumar-sahu**
- Added home, signIn and signUp pages.
- Added routes for the pages.
- Live link --> https://deploy-preview-1--shop-up-react.netlify.app/

### CODE REVIEW

File: front-end/src/components/common/navbar.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { FontAwesomeIcon } from './fontAwesomeIcon';
+
+export const Navbar = () => {
+	return (
+		<nav id='navbar' className='navigation-hovered flex-j-space-a-center p-5 mb-3'>
+			<div className='flex-align-center'>
+				<FontAwesomeIcon iconStyle={'fa fa-bars list-item__headerIcon mr-4'} />
+				<input
+					className='navigation-hovered__search'
```
Comment (ajit-sharmaGH) [inline front-end/src/components/common/navbar.js:9]: Good naming convention

File: front-end/src/components/common/navbar.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+					type='search'
+					placeholder='Search...'
+				/>
+			</div>
+			<div className='flex-row'>
+				<FontAwesomeIcon iconStyle={'fas fa-heart list-item__headerIcon mr-4'} />
+				<FontAwesomeIcon iconStyle={'fas fa-cart-plus list-item__headerIcon mr-4'} />
```
Comment (ajit-sharmaGH) [inline front-end/src/components/common/navbar.js:16]: nicely used utility classes

**Review by ajit-sharmaGH — APPROVED** [review]
Looks good 
You can merge it
Good naming convention
Easy to read

File: front-end/src/components/common/navbar.js (hunk 2 above)
Comment (abhijeetekad) [reply front-end/src/components/common/navbar.js:16]: +1

File: front-end/src/components/common/productCard.js [hunk 3]
```diff
@@ -44,10 +31,10 @@ export const ProductCard = () => {
 							₹ 60990/-
 						</div>
 					</div>
-					<div className='mb-2'>
-						<FontAwesomeIcon iconStyle={'far fa-heart'} otherStyle={'mr-2'} />
+					{/* <div className='mb-2'>
+						<FontAwesomeIcon iconStyle={'far fa-heart mr-2'} />
 						<FontAwesomeIcon iconStyle={'fas fa-shopping-cart'} />
-					</div>
+					</div> */}
```
Comment (abhijeetekad) [inline front-end/src/components/common/productCard.js:37]: Remove commented code if not required

**Review by abhijeetekad — APPROVED** [review]
Code is well structured, clean and easy to understand. Just one suggestion add database and then render data instead of manually using components.

**Comment by pratik-kumar-sahu** [comment]
> Code is well structured, clean and easy to understand. Just one suggestion add database and then render data instead of manually using components.

**Pull request merged**